profile/README.md

```diff
@@ -39,12 +39,12 @@ Our team of dedicated experts is here to help groups and individuals who want to
 Lombiq's team consists of highly skilled experts from diverse backgrounds:
 
  
- <img src="lombiq-team-2021-summer_50.jpg" alt="lombiq team 2021 summer" width="1000"  height="700">
+ ![lombiq team 2021 summer](lombiq-team-2021-summer.jpg) 
```

Comment:
Again, please use correct spelling in the alt text. The company's name is not "lombiq".